app/src/main/java/org/mozilla/tv/firefox/pocket/PocketVideoFetchScheduler.kt

```diff
@@ -53,6 +53,21 @@ class PocketVideoFetchScheduler(
             .setRequiredNetworkType(NetworkType.CONNECTED)
             .build()
 
+        // This sets the custom fetch delay for testing purposes.
+        // The custom delay will only run once.
+        val delay: Long
+        val workPolicy: ExistingWorkPolicy
+
+        if (IS_QA_BUILD) {
+            delay = checkNotNull(DELAY_TIME_FOR_QA_MILLIS) { "Fetch delay value must be set" }
+            workPolicy = ExistingWorkPolicy.REPLACE
+            // Only allow this value to be used once so we do not end up spamming the Pocket servers
+            IS_QA_BUILD = false
```

Comment:
We should null out the delayMillis too, right? Otherwise we're being inconsistent how we handle the state.

It might be good to extract this into a "reset" helper function that is next to `setDelayForQA` so that it's visible what states are intended to be set and unset together.